mobile/ios/Runner/Info.plist

```diff
@@ -1,105 +1,97 @@
+		<string>Immich</string>
+		<key>CFBundleExecutable</key>
+		<string>$(EXECUTABLE_NAME)</string>
+		<key>CFBundleIdentifier</key>
+		<string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
+		<key>CFBundleInfoDictionaryVersion</key>
+		<string>6.0</string>
+		<key>CFBundleName</key>
+		<string>immich_mobile</string>
+		<key>CFBundlePackageType</key>
+		<string>APPL</string>
+		<key>CFBundleShortVersionString</key>
+		<string>1.42.0</string>
+		<key>CFBundleSignature</key>
+		<string>????</string>
+		<key>CFBundleVersion</key>
+		<string>79</string>
+		<key>LSRequiresIPhoneOS</key>
+		<true/>
+		<key>MGLMapboxMetricsEnabledSettingShownInApp</key>
+		<true/>
+		<key>NSAppTransportSecurity</key>
+		<dict>
+			<key>NSAllowsArbitraryLoads</key>
+			<true/>
+		</dict>
+		<key>NSLocationAlwaysUsageDescription</key>
+		<string>Enable location setting to show position of assets on map</string>
+		<key>NSLocationWhenInUseUsageDescription</key>
+		<string>Enable location setting to show position of assets on map</string>
+		<key>NSPhotoLibraryUsageDescription</key>
+		<string>We need to manage backup your photos album</string>
+		<key>NSPhotoLibraryAddUsageDescription</key>
+		<string>We need to manage backup your photos album</string>
+		<key>NSCameraUsageDescription</key>
+		<string>We need to access the camera to let you take beautiful video using this app</string>
+		<key>NSMicrophoneUsageDescription</key>
+		<string>We need to access the microphone to let you take beautiful video using this app</string>
+		<key>UILaunchStoryboardName</key>
+		<string>LaunchScreen</string>
+		<key>UIMainStoryboardFile</key>
+		<string>Main</string>
+		<key>UISupportedInterfaceOrientations</key>
+		<array>
+			<string>UIInterfaceOrientationPortrait</string>
+			<string>UIInterfaceOrientationLandscapeLeft</string>
+			<string>UIInterfaceOrientationLandscapeRight</string>
+		</array>
+		<key>UISupportedInterfaceOrientations~ipad</key>
+		<array>
+			<string>UIInterfaceOrientationPortrait</string>
+			<string>UIInterfaceOrientationPortraitUpsideDown</string>
+			<string>UIInterfaceOrientationLandscapeLeft</string>
+			<string>UIInterfaceOrientationLandscapeRight</string>
+		</array>
+		<key>UIViewControllerBasedStatusBarAppearance</key>
+		<true/>
+		<key>io.flutter.embedded_views_preview</key>
+		<true/>
+		<key>ITSAppUsesNonExemptEncryption</key>
+		<false/>
+		<key>CADisableMinimumFrameDurationOnPhone</key>
+		<true/>
+		<key>LSApplicationQueriesSchemes</key>
+		<array>
+			<string>https</string>
+		</array>
+		<key>CFBundleLocalizations</key>
+		<array>
+			<string>cs</string>
+			<string>da</string>
+			<string>de</string>
+			<string>en</string>
+			<string>es</string>
+			<string>fi</string>
+			<string>fr</string>
+			<string>it</string>
+			<string>ja</string>
+			<string>ko</string>
+			<string>nl</string>
+			<string>pl</string>
+			<string>pt</string>
+			<string>ru</string>
+			<string>sk</string>
+			<string>zh</string>
+		</array>
+		<key>UIStatusBarHidden</key>
```

Comment:
Is this auto generated by some process in this PR?

Reply:
Odd, it might be. Let me take a look.

Reply:
Yup, I just reran the script `flutter pub run flutter_native_splash:create` (from the `makefile`) and this line shows up in the output:

`[iOS] Updating ios/Runner/Info.plist for status bar hidden/visible`

I suppose that answers our question. It sure generates a terrible diff. I think it's just an indentation, but I didn't read it very carefully. The main relevant line it seems to be adding is here at the end:

```
<key>UIStatusBarHidden</key>
<false/>
```